tests/PHPUnit/Framework/TestCase/IntegrationTestCase.php

```diff
@@ -81,6 +82,8 @@ public function setUp()
 
         PiwikCache::getEagerCache()->flushAll();
         PiwikCache::getTransientCache()->flushAll();
+
+        Translate::loadAllTranslations();
```

Comment:
That makes a whole lot of integration tests fail. Many of them are written to test against translation keys, which is good because if the translation change the test still pass. So I don't think we should load all translations for all tests.